token/program2/src/state.rs

```diff
@@ -36,16 +38,42 @@ pub struct Account {
     /// If `delegate` is `Some` then `delegated_amount` represents
     /// the amount authorized by the delegate
     pub delegate: COption<Pubkey>,
-    /// Is `true` if this structure has been initialized
-    pub is_initialized: bool,
+    /// Whether the account has been initialized or frozen
+    pub is_initialized: AccountState,
```

Comment:
Perhaps?
```suggestion
    pub account_state: AccountState,
```

Reply:
And `s/Whether the account has been initialized or frozen/The account's state/` or something like that